src/components/Common/BadgeMobile/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { forwardRef } from 'react';
+
+interface BadgeMobileProps {
+  badgeType?:
+    | 'default'
+    | 'primary'
+    | 'primary4K'
+    | 'danger'
+    | 'danger4K'
+    | 'warning'
+    | 'warning4K'
+    | 'success'
+    | 'success4K';
+  className?: string;
+  href?: string;
+  children: React.ReactNode;
+}
+
+const BadgeMobile = (
```

Comment:
Let's name this StatusBadgeMobile instead and refactor it to handle rendering the icons as well.